pkg/clients/openshift.go

```diff
@@ -288,3 +330,56 @@ func (o OpenshiftClient) SubjectRulesReview(user, namespace string, log *logging
 	}
 	return authorization.ConvertAPIPolicyRulesToRBACPolicyRules(pr), nil
 }
+
+// ConvertRegistryImagesToSpecs - Return APB specs from internal OCP registry
+func (o OpenshiftClient) ConvertRegistryImagesToSpecs(log *logging.Logger, imageList []string) ([]FQImage, error) {
+	var fqList []FQImage
+	fqImage := FQImage{}
+	var err error
+	r := &ImageList{}
+	err = o.imageRestClient.Get().
```

Comment:
Sorry, didn't catch this one before, but should check the error before working with the items.

Reply:
Thanks, fixed